spec/hatchet/config_spec.rb

```diff
@@ -6,6 +6,10 @@
     expect(@config.path_for_name("rails3_mri_193")).to(eq("repo_fixtures/repos/rails3/rails3_mri_193"))
   end
 
+  it("config path for name with full repo name") do
```

Comment:
Style nit. The rspec methods don't use parens `it` `describe` etc. It reads fine, not enough to block merging. But it did stand out.

We could consider adding a style linter like standard-rb/prettier/rubocop. Setting them up is always a pain though. The smaller the team that modifies the source, the less value linters provide IMHO.

Reply:
Ah I missed there were parens. I'd copied/pasted the block above, which also has parens.

I have rubocop + the rspec rubocop plugin enabled for the Python repo. I found the onbaording experience excellent with Rubocop compared to other linters -- I used the auto-generated "todos" file + autocorrect to get most the way there, then permanently disabled any cops that were still failing and didn't seem to add value.

Reply:
Ahh. Originally the tests were written in minitest. But the ecosystem around plugins (i.e. parallel testing) isn't as robust as rspec. I used a tool to auto-convert one test syntax to the other. I think it added the parens.